### DISCUSSION TITLE
[IDEA] Paste an image into excalidraw, turned into excalidraw-style drawing, also text to image (DALLE)

### DISCUSSION THREAD

**Discussion by louis030195** [excalidraw/excalidraw#3990] [General]
Say I copy an image of Clubhouse app from Google Image because I want to make a drawing inspired from it:

![image](https://user-images.githubusercontent.com/25003283/134195815-77327476-04d8-4769-a4c4-093cd56546bf.png)

I paste into excalidraw, use probably some ML model to infer the UI using excalidraw components.

End up looking to something like:

![image](https://user-images.githubusercontent.com/25003283/134196406-ccf14fa8-3ef6-46de-b2ba-c366fb5dfb18.png)

Another thing that could help is using something like https://huggingface.co/spaces/flax-community/dalle-mini trained on excalidraw, so you can say "draw me a clubhouse app" and it would draw some basic excalidraw clubhouse and from that I can start improving fast

**Comment by AnimeshRy** [comment]
Nice idea

**Comment by Noahbest6** [comment]
i want this

**Comment by Mrazator** [comment]
Let's track it here #9244
